eng/versioning/version_client.txt

```diff
@@ -1,12 +1,10 @@
 # Format;
 # <groupId>:<artifactId>;dependency-version;current-version
 
-com.azure:azure-sdk-all;1.0.0;1.0.0
-com.azure:azure-sdk-parent;1.6.0;1.6.0
-com.azure:azure-client-sdk-parent;1.7.0;1.7.0
+com.azure:azure-active-directory-spring-boot-starter;2.3.2;2.3.3-beta.1
```

Comment:
@chenrujun, I'm going to recommend that you completely revert this file and start from scratch for the following reasons:
1. Please do not re-organize this entire file. These top 3 are ancestor poms for the track and the repository. Everything under the com.azure:azure-client-sdk-parent entry should be alphabetized (since they're actually shipping libraries), but if something isn't it's probably because the name changed and it wasn't reordered and I'll take care of that at a later time. It only makes the PR more confusing and harder to read.
2. The current groupId for your spring libraries is com.microsoft.azure, not com.azure. You've added a number of duplicate spring boot artifact entries with the wrong groupId.
3. The way versions are incremented post release are as follows:
  * If the release was beta then beta.x becomes beta.x+1
  * If the release was GA, then we zero out the hotfix, increment the minor and add "-beta.1" to the end.

Reply:
Updated.

Reply:
Hi @JimSuplizio .

> The way versions are incremented post release are as follows:
> If the release was beta then beta.x becomes beta.x+1
> If the release was GA, then we zero out the hotfix, increment the minor and add "-beta.1" to the end.

We have already released 2.3.2, can I write version like this?

```
com.microsoft.azure:azure-spring-boot;2.3.2;2.3.3-beta.1
```

Reply:
I'm going to let this through but I was pretty specific in about what the version increment should have been. After releasing 2.3.2, the new 'current' should be 2.4.0-beta.1. This is the way we increment things after release and the way the auto increment code will work when it creates the auto increment PR when released through our pipelines.

Reply:
I got it. Thank you for your detailed explanation.